lib/Doctrine/MongoDB/EagerCursor.php

```diff
@@ -219,6 +230,54 @@ public function fields(array $f)
     }
 
     /**
+     * {@inheritDoc}
+     */
+    public function getCollection()
+    {
+        return $this->cursor->getCollection();
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function getConnection()
+    {
+        return $this->cursor->getConnection();
+    }
```

Comment:
If we remove this method from the interface (because it's deprecated, I suppose we don't need to add it here); however, if you'd like to keep it in the interface, by all means leave it.